Sources/SwiftDocCUtilities/Action/Actions/Convert/ConvertAction.swift

```diff
@@ -483,15 +483,39 @@ public struct ConvertAction: Action, RecreatingContext {
         }
 
         var didEncounterError = analysisProblems.containsErrors || conversionProblems.containsErrors
-        if try context.renderRootModules.isEmpty {
+        let hasTutorial = context.knownPages.contains(where: {
+            guard let kind = try? context.entity(with: $0).kind else { return false }
+            return kind == .tutorial || kind == .tutorialArticle
+        })
+        // Warn the user if the catalog is a tutorial but does not contains a table of contents
+        // and provide template content to fix this problem.
+        if (
+            context.rootTechnologies.isEmpty &&
+            hasTutorial &&
+            !analysisProblems.contains(where: {$0.diagnostic.identifier == "org.swift.docc.HasExactlyOne<Tutorials, Intro>.Missing"})
+        ) {
+            let tableOfContentsFilename = "table-of-contents.tutorial"
+            let source = rootURL?.appendingPathComponent(tableOfContentsFilename)
             postConversionProblems.append(
                 Problem(
                     diagnostic: Diagnostic(
+                        source: source,
                         severity: .warning,
-                        identifier: "org.swift.docc.MissingTechnologyRoot",
-                         summary: "No TechnologyRoot to organize article-only documentation.",
-                         explanation: "Article-only documentation needs a TechnologyRoot page (indicated by a `TechnologyRoot` directive within a `Metadata` directive) to define the root of the documentation hierarchy."
-                     )
+                        identifier: "org.swift.docc.MissingTableOfContents",
+                        summary: "Missing tutorial table of contents page.",
+                        explanation: "`@Tutorial` and `@Article` pages require a `@Tutorials` table of content page to define the documentation hierarchy."
```

Comment:
I'm not entirely sure I understand this 🤔. Are you suggesting adding a note to the warning indicating that they are converting a tutorial catalog?

If the user receives this warning, it's because they are attempting to convert a `.tutorial` file. Additionally, the `@Article` directive is only supported in tutorials. Therefore, by this point, the user may already realize that these are tutorial-related directives.

What additional clarification do you believe is necessary?